src/utils/resolve.js

```diff
@@ -15,6 +16,10 @@ export default function resolve(css, styles) {
     inlineStyles,
   } = separateStyles(flattenedStyles);
 
+  aphroditeStyles.forEach((stylesObj) => {
```

Comment:
This is so that `cssNoRTL` works on a style definition, even after a regular with rtl resolve has been called.

Reply:
I wonder if this could be moved into `separateStyles` so that we can avoid an extra iteration. Perhaps `separateStyles` could accept a function as an argument which would allow each interface to pass in something to do arbitrary processing on each iteration.